src/nulog-inference-service/build.sh

```diff
@@ -1,4 +1,6 @@
+#IMAGE_NAME=tybalex/nulog-inference-service-control-plane:v0.0
```

Comment:
So the same build.sh file is used to build the nulog-inference-service-control-plane and nulog-inference-service images?

Reply:
yes, the codes are same. Only difference is, in `nulog-CP` image, we also include a pre-train CP model.